fog/sql_recovery_db/src/lib.rs

```diff
@@ -263,6 +264,44 @@ impl SqlRecoveryDb {
             .map(|val| val as u64))
     }
 
+    fn get_expired_invocations_impl(
+        &self,
+        conn: &PgConnection,
+        expiration: NaiveDateTime,
+    ) -> Result<Vec<ExpiredInvocationRecord>, Error> {
+        use schema::ingest_invocations::dsl;
+        let query = schema::ingest_invocations::dsl::ingest_invocations
+            .select((
+                dsl::id,
+                dsl::rng_version,
+                dsl::egress_public_key,
+                dsl::last_active_at,
+            ))
+            .filter(dsl::last_active_at.lt(expiration));
```

Comment:
:x: I'm not 100% that this is an issue and it's not necessarily in this PR.

I'm fighting getting the tests running locally to better understand how this works...

Looking at line 481
```rust
            let now =
                diesel::select(diesel::dsl::now).get_result::<chrono::NaiveDateTime>(&conn)?;
```

Looking at the diesel docs for [`now`](http://docs.diesel.rs/diesel/dsl/struct.now.html) it stats
> Represents the SQL CURRENT_TIMESTAMP constant. This is equivalent to the NOW() function on backends that support it.

Then looking at the postgresql docs for [`now()`](https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT) it says
> now() is a traditional PostgreSQL equivalent to transaction_timestamp().

and
> transaction_timestamp() is equivalent to CURRENT_TIMESTAMP

and
> CURRENT_TIME and CURRENT_TIMESTAMP deliver values with time zone;

So back to line 481.  It takes now and converts it to [`NaiveDateTime`](https://docs.diesel.rs/master/chrono/naive/struct.NaiveDateTime.html) which says
> They have almost equivalent interfaces as their timezone-aware twins, but are not associated to time zones 

It's not clear if `now()` -> `NativeDateTime` will be normalized to UTC, from the provided docs, I don't think it will be...